File: result/palindrome_string.py
a = [None] * 100
def main():
	global a
	num = None
	print("Input the string length: ")
	num = eval(input())
	print("Input string: ")
	a = input()
	half = (num / 2)
	i = 0
	while (i < half):
		x = (num - 1)
		a1 = a[i]
		a2 = a[(x - i)]
		if (a1 != a2):
			print("FALSE!")
			return 0
		i = (i + 1)
	print("TRUE!")
	return 0

if __name__ == '__main__':
	main()

File: exe/palindrome_string.c
char a[100];

int main() {
    int num;
    printf("Input the string length: ");
    scanf("%d", &num);
    printf("Input string: ");
    scanf("%s", a);
    int half = num / 2;
    for (int i = 0; i < half; i = i + 1) {
        int x = num - 1;
        int a1 = a[i];
        int a2 = a[x - i];
        if (a1 != a2) {
            printf("FALSE!");
            return 0;
        }
    }
    printf("TRUE!");
    return 0;
}